cylc/flow/scripts/remove.py

```diff
@@ -18,7 +18,26 @@
 
 """cylc remove [OPTIONS] ARGS
 
-Remove one or more task instances from a running workflow.
+Remove task instances from a running workflow and the workflow's history.
+
+This removes the task(s) from any specified flows. The task will still exist,
+just not in the specified flows, so will not influence the evolution of
+the workflow in those flows.
+
+If a task is removed from all flows, it and its outputs will be left in the
+`None` flow. This preserves a record that the task ran, but it will not
+influence any flows in any way.
```

Comment:
```suggestion
If a task is removed from all flows, it and its outputs will be left in the
`None` flow to preserve a record that the task ran.
```
The last bit (it will not influence ...) is already stated in the previous paragraph.

Reply:
Now outdated.